### ISSUE TITLE
Can I create my own browser image

### ISSUE THREAD

**Issue by Believe1011** [m1k1o/neko-rooms#125]
Can I create my own browser image? I would like to make some configurations in the browser image, such as setting the browsing language, adding Google plugins, etc

**Comment by m1k1o** [comment]
Yes, go to neko repository and folder .docker for instructions.

When you have your image built, make sure not to pull that image from the remote registry, because it would overwrite your changes.

**Comment by Believe1011** [comment]
I want to add default plugins to Google Chrome when building a Google Chrome image. I have already downloaded the plugins in advance and placed them in the specified directory during the build process. However, when neko rooms started using this image, the plugins were not installed and I used Docker Exec to check if there were any plugins I added. What should I do to correctly add plugins when building plugins


[Dockerfile.txt](https://github.com/m1k1o/neko-rooms/files/15128053/Dockerfile.txt)
